### ISSUE TITLE
attr_accessor_initialize ?

### ISSUE THREAD

**Issue by nitzanav** [barsoom/attr_extras#18]
Hi,

I love attr_extras, I use it everywhere, and chose it the tenth of alternative, great work guys.

Though I have case that I need reader, writer and initializer. sometimes private, public or protected.

So why not enabling something like:
```ruby
attr_accessor_initialize :foo, :goo
```

Instead of this:
```ruby
attr_initialize :foo, :goo
attr_accessor :foo, :goo
```
or this:
```ruby
attr_reader_initialize :foo, :goo
attr_writer :foo, :goo
```

**Comment by henrik** [comment]
Hey, thanks for the feedback!

That sounds like a reasonable addition. What would be a good short name for it? `aattr_initialize`? Too similar?

**Comment by nitzanav** [comment]
I do think that aattr_initialize is the right short name because it follows you convention in the best way.

I personally use the long names as they are more readable. Your methods are doing so much in so little code. Hence I don't see any reason to save 3 more characters as I saved so much already.

**Comment by henrik** [comment]
Sounds fair. Alright, if someone wants to make a PR for this it would be great. If not I'll leave this on my todo list and hope to get to it some day :)

**Comment by henrik** [comment]
Closed by PR #21.
